### PR TITLE
Updated tweet_historical_events.py. Added historical event. 

### PR DESCRIPTION
**Pull request by gremlin97**
This contributes to #54

### CODE REVIEW

**Review by bobsany16 — APPROVED** [review]

File: words_dir/tweet_nouns.py [hunk 1]
```diff
@@ -347,7 +347,17 @@
...
     'aurora',
-
+    'meadow',
+    'rainforest',
+    'volcano',
+    'petrichor',
+    'cliff',
+    'coast',
+    'cave',
+    'desert'
```
Comment (bobsany16) [inline words_dir/tweet_nouns.py]: could you add the missing comma ? 😁

Comment (gremlin97) [reply words_dir/tweet_nouns.py]: Sorry :disappointed: Working on it!

**Comment by gremlin97** [comment]
This contributes to #3

File: words_dir/tweet_nouns.py [hunk 2]
```diff
@@ -347,7 +347,17 @@
...
     'aurora',
-
+    'meadow',
+    'rainforest',
+    'volcano',
+    'petrichor',
+    'cliff',
+    'coast',
+    'cave',
+    'desert,
```
Comment (bobsany16) [inline words_dir/tweet_nouns.py]: @gremlin97 it's okay, it happens to the best of us! We live and we learn, right ? Don't worry about it. Also, I don't mean to be nit-picky but it's just missing a quotation mark now. One more fix 💪

Comment (gremlin97) [reply words_dir/tweet_nouns.py]: :rofl: I will tripple check this time. Thanks for the motivation :v: and sorry for the trouble!

**Review by VickiLanger — APPROVED** [review]
Thanks for your contribution. 

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/LGBTQotd) to see the hard work you put in.*

**Pull request merged**